esp-hal/README.md

```diff
@@ -46,6 +46,63 @@ For help getting started with this HAL, please refer to [The Rust on ESP Book] a
 [s2-trm]: https://www.espressif.com/sites/default/files/documentation/esp32-s2_technical_reference_manual_en.pdf
 [s3-trm]: https://www.espressif.com/sites/default/files/documentation/esp32-s3_technical_reference_manual_en.pdf
 
+## Peripheral support
+
+<!-- start chip support table -->
```

Comment:
We should mention this table is machine generated in a comment so that users don't try and manually modify this.